src/main/java/org/jabref/gui/keyboard/KeyBindingRepository.java

```diff
@@ -142,6 +142,7 @@ public KeyStroke getKey(KeyBinding bindName) {
 
     private KeyCombination getKeyCombination(KeyBinding bindName) {
         String binding = get(bindName.getConstant());
+        binding = binding.replaceAll(" ", "+"); //javafdx expects plus signs between modifiers, swing not
```

Comment:
I would actually prefer it the other way around: store it in JavaFX format and replace "+" -> " " for swing since we want to completely switch to FX at some point.